cypress/e2e/gui/testsGui.cy.js

```diff
@@ -0,0 +1,358 @@
+describe('Validate the GUI', () => {
+  context('Validate the no customers scenarios', () => {
+    beforeEach(() => {
+      const CUSTOMERS_API_URL = `${Cypress.env('API_URL')}/customers`;
+      cy.intercept(
+        'GET',
+        `${CUSTOMERS_API_URL}?**`,
+        {
+          'customers': [],
+          'pageInfo': {
+            'currentPage': 1,
+            'totalPages': 1,
+            'totalCustomers': 50
+          }
+        }
+      ).as('getNoCustomers');
+      cy.setCookie('cookieConsent', 'accepted');
+      cy.visit('/');
+    });
+    it('The empty box image should be visible', () => {
+      cy.get('[title="image of an empty box"]').should('be.visible');
+    });
+    it('The input name should be disabled', () => {
+      cy.get('[data-testid="name"]')
+        .should('be.visible')
+        .and('have.attr', 'disabled');
+    });
+  });
+
+  context('Validate the customer missing info scenarios', () => {
+    beforeEach(() => {
+      const CUSTOMERS_API_URL = `${Cypress.env('API_URL')}/customers`;
+      cy.intercept(
+        'GET',
+        `${CUSTOMERS_API_URL}?**`,
+        { fixture: 'customerMissingInfo.json' }
```

Comment:
```suggestion
        { fixture: 'customerMissingInfo' }
```

A extenção pode ser omitida e o Cypress saberá lidar com isso.

Isso torna a linha mais curta e um pouco mais simples.